extras/rapidfuzz_amalgamated.hpp

```diff
@@ -2639,7 +2639,7 @@ std::size_t levenshtein_myers1999_block(basic_string_view<CharT1> s2,
       const uint64_t PbTemp = Pb;
       Pb = Ph >> 63;
       Ph = (Ph << 1) | PbTemp;
-    
```

Comment:
could you remove those trailing whitespaces in `rapidfuzz/details/string_metrics/levenshtein_impl.hpp` as well. 
rapidfuzz_almagated.hpp is a file generated by `tools/amalgamation.py`, so unless they are removed in there I would reintroduce them when I generate the file the next time.

Reply:
Yes, done!